scapy/data.py

```diff
@@ -175,27 +175,37 @@ def load_manuf(filename):
                 else:
                     lng = l[i+2:]
                 manufdb[oui] = shrt, lng
-            except Exception as e:
-                log_loading.warning("Couldn't parse one line from [%s] [%r] (%s)" % (filename, l, e))
-    except IOError:
-        log_loading.warning("Couldn't open [%s] file" % filename)
-        return ""
+            except Exception:
+                log_loading.warning("Couldn't parse one line from [%s] [%r]",
+                                    filename, l, exc_info=True)
     return manufdb
     
 
-
 if WINDOWS:
     ETHER_TYPES=load_ethertypes("ethertypes")
     IP_PROTOS=load_protocols(os.environ["SystemRoot"]+"\system32\drivers\etc\protocol")
     TCP_SERVICES,UDP_SERVICES=load_services(os.environ["SystemRoot"] + "\system32\drivers\etc\services")
     # Default value, will be updated by arch.windows
-    MANUFDB = load_manuf(os.environ["ProgramFiles"] + "\\wireshark\\manuf")
+    try:
+        MANUFDB = load_manuf(os.environ["ProgramFiles"] + "\\wireshark\\manuf")
+    except IOError:
+        log_loading.warning("Cannot read [%s]", filename)
+        MANUFDB = None
```

Comment:
The log_loading is confusing, as MANUFDB is updated inside arch/windows if not available, when wireshark is detected...